python/sanlock.c

```diff
@@ -943,6 +1004,7 @@ py_get_hosts(PyObject *self __unused, PyObject *args, PyObject *keywds)
     ls_list = __hosts_to_list(hss, hss_count);
 
 exit_fail:
```

Comment:
For consistency, we should call this "finally" - this is not a failure path. Since this not an issue added by the patch we can fix it later.